devops/setup-gpg/action.yaml

```diff
@@ -1,52 +1,72 @@
 name: 'Setup GPG'
 description: 'Configures this action to run gpg with a given key and pass'
 inputs:
-  gpg-private-key:  # id of input
+  gpg-private-key:
+    description: 'GPG private key exported as an ASCII armored version or its base64 encoding'
     required: true
-  gpg-key-pass:  # id of input
+  gpg-key-pass:
+    description: 'Passphrase of the GPG private key'
     required: true
-  gpg-key-name:  # id of input
+  gpg-public-key:
+    description: 'GPG public key exported as an ASCII armored version or its base64 encoding'
     required: true
-    default: "Aerospike"
+  gpg-key-name:
+    description: 'GPG key name'
+    required: true
+    default: "aerospike-inc"
+  gpg-trust-level:
+    description: 'Set key trust level'
+    required: false
+    default: 5
 runs:
   using: "composite"
   steps:
-      - name: "check if private key is not empty"
+      - name: "check if private key is empty"
         env: 
-            PRIVATE_KEY: ${{ inputs.gpg-private-key }}
-        if: ${{ env.PRIVATE_KEY == '' }}
+            GPG_PRIVATE_KEY: ${{ inputs.gpg-private-key }}
+        if: ${{ env.GPG_PRIVATE_KEY == '' }}
+        run: |
+          echo "The gpg-private-key was empty"
+          exit 1
+        shell: bash
+      - name: "check if public key is empty"
+        env:
+            GPG_PUBLIC_KEY: ${{ inputs.gpg-public-key }}
+        if: ${{ env.GPG_PUBLIC_KEY == '' }}
         run: |
-          echo "the gpg-private-key was empty"
+          echo "The gpg-public-key was empty"
           exit 1
         shell: bash
-      - name: "check if key name is not empty"
+      - name: "check if key name is empty"
         env: 
-            KEY_NAME: ${{ inputs.gpg-key-name }}
-        if: ${{ env.KEY_NAME == '' }}
+            GPG_ID: ${{ inputs.gpg-key-name }}
```

Comment:
I'm not sure why we renamed this. We should ether leave it KEY_NAME or change everything else to GPG_ID ether is fine, but I would say we make that a seperate PR as it's conflating two issues

Reply:
I don't change this as all.  This is from your original code.

Reply:
I was just saying to remove it for the "important stuff" it's fine and I understand what and why your where doing, it's just doing a bunch of things instead of one thing. I'm not even sure of the "issue" that you solved with this update! :)